RoomMe/app/src/main/java/com/example/trocket/roomme/EditProfileView.java

```diff
@@ -0,0 +1,38 @@
+package com.example.trocket.roomme;
+
+import android.support.v7.app.ActionBarActivity;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+
+
+public class EditProfileView extends ActionBarActivity {
```

Comment:
Oh poop. Ok, I forgot about this. Every list item on the navigation drawer (ex: edit profile, RoomMe List, settings, etc.) will need to be a Fragment rather than an Activity. This is because when we click on, say, Edit Profile, on the nav drawer, we're not switching activities, we're overlaying a fragment on top of the main activity. A fragment is very similar to an activity. So do the following:

1) change "extends ActionBarActivity" to "extends Fragment"
2) implement the necessary method (onCreateView)

See the following site (look at #13): https://www.codeofaninja.com/2014/02/android-navigation-drawer-example.html

Let me know if you need help. Sorry about the late notice. Forgot about this when creating the nav drawer